source/process/docs-reviewers-guide.rst

```diff
@@ -0,0 +1,116 @@
+Reviewer's Guide for docs.mattermost.com 
+========================================
+
+Review of pull requests to Mattermost documentation passes through the following high-level steps: 
+
+1. Project Members and Approved Contributors submit PRs to the docs.mattermost.com repository. 
+2. Incoming PRs are assessed and assigned review requirements (e.g. technical PRs, PRs requiring Sphinx changes, etc.). 
+3. Reviewers with appropriate qualifications either approve and merge the changes, or leave feedback for the submitter to begin the review cycle again. 
+4. Once merged, PRs are verified by a Project Member and the submitter's work is acknowledged. 
+5. Reviewers with consistent performance are acknowledged with an increase in `skill level <link to below>`_. Reviewers with repeated inconsistency may have their level decreased (though this is rare). 
+
+The following sections lay out the details for this process: 
+
+Pre-reading 
+-------------
+
+- Please read `Contributor's Guide for docs.mattermost.com XXXXXX <XXXX>`_ for the process by which Approved Contributors submit PRs and engage with reviewers. 
+
+Review Labels 
+-------------
+
+The following labels are used to indicate the type of review required for each doc PR: 
+
+- **Needs Style Guide Approval** - Review for `style guide requirements <https://docs.mattermost.com/process/sg_mattermost-doc-style.html>`_, and specifically: 
+
+   - Verify all links work
+   - Verify grammar is correct based on US English requirements 
+   - Verify RST renders properly in GitHub preview
+
+- **Needs Technical Approval** - Technical change requiring developer review, e.g. change in system requirements.
+
+- **Needs Testing Approval** - Change requires manual testing, e.g. significant install process change. 
+
+- **Needs Sphinx Approval** - Significant structural changes to CSS or Sphinx features, requires HTML generation and verification. 
+
+- **Needs Structural Approval** - Approval required on changes to TOC or organization.
+
+- **Needs Verification & Acknowledgement** - docs.mattermost.com needs to be verified and acknowledgement added to PR.
+
+- **Post-Merge Editing Required** - PR can be merged and someone will follow-up with edits for minor corrections.
+
+- **Awaiting Submitter Action** - Merge should not happen until feedback is addressed. Note: This label should be applied for any work-in-progress PR, a separate label is not needed. 
+
```

Comment:
I think the labels might be too granular. I think if you create roles such as Reviewer, Technical Reviewer, and Editor, the people with those roles should be responsible for making sure that, for example, the changes are tested or the Sphinx output is verified. And then the labels would be of a different nature... "Review Required", "Technical Review Required", "Copy Edit Required", and the like.